source/standards/threat-modelling.html.md.erb

```diff
@@ -0,0 +1,236 @@
+---
+title: Threat Modelling
+last_reviewed_on: 2021-06-25
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## What's a threat?
+
+A threat is an action that causes denial, alteration or disclosure of your
+assets that you are trying to protect. It could be a user database, a deployment
+pipeline or the integrity of web form submissions.
+
+## What's threat modelling?
+
+Threat modelling is analysing a service to raise potential information security
+and privacy issues.
+
+The best time to perform threat modelling activities is during the design phase;
+however, it can be done anytime and should become a continuous process in your
+service team.
+
+Within the Cabinet Office and GDS, we follow the [Threat Modeling Manifesto]'s
+four questions:
+
+1. What are we working on?
+2. What can go wrong?
+3. What are we going to do about it?
+4. Did we do a good enough job?
+
+We answer most of the above over three sessions, with time between allowing for
+asynchronous input.
+
+### 1. Threat Discovery
+
+Threat discovery is looking to answer the “What are we working on?” and part of
+the “What can go wrong?” question.
+
+We use an interactive exercise to talk through and explain the system or
+architecture. Someone from the service team will need to lead this part. Try to
+keep the scope small; it may be more effective to break a service down and
+perform multiple threat modelling activities.
+
+The group of people involved typically includes the development team, cyber
+security and information assurance colleagues; however, working with people in
+the product and user research space often provides more varied perspectives.
+
+As part of the interactive exercise, we surface hypothetical threats or
+vulnerabilities in a service. We use [STRIDE] to help us look at different
+aspects and ensure a rounded view of all threats.
+
+#### 1.1 Interactive Exercise
+
+The purpose of this exercise is to talk through the service, architecture or the
+user journey with people in a session, allowing for questions and answers.
+
+In-person, diagrams are typically drawn up on a whiteboard, followed by
+questions and post-its of potential threats stuck at relevant points.
+
+Remotely, [Jamboard] can be used to add virtual post-its on top of the digital
+diagrams.
+
+### 2. Threat Analysis
+
+Threat analysis looks to finalise the answer to the “What can go wrong?”
+question. We use a scoring methodology to determine if a threat is valid and
+prioritise them against each other.
+
+#### 2.1 Scoring
+
+After discovery, you can copy a Google Sheet template to use for noting down
+threats, the most likely actor, scoring, and tracking mitigation.
+
+The template sheet with notes is here:
```

Comment:
Could we compress this step into the one above? For example, 'after the discovery stage, you can make a copy of the [Threat Modelling Scoring template](https://docs.google.com/spreadsheets/d/1u22W_bUEPESvbMde-Q4syJLTen1OKIcE4ILk7wyaydM/edit#gid=0) to use for noting down...'